gcs/gcs.go

```diff
@@ -333,7 +333,25 @@ func (f *Filter) Match(key [KeySize]byte, data []byte) (bool, error) {
 // probability) to be a member of the set represented by the filter faster than
 // calling Match() for each value individually.
 func (f *Filter) MatchAny(key [KeySize]byte, data [][]byte) (bool, error) {
-	// Basic sanity check.
+	// TODO(conner): add real heuristics to query optimization
+	switch {
+
+	case len(data) >= int(f.N()/2):
```

Comment:
So `N` here is actually the number of bytes, and not necessarily the number of elements as it's about 3 bit per element or so as is.

Reply:
turns out N is the number of elements, so i've left the heuristics in tact. did realize that the bench mark was using 2000 instead of 5000 so i went back and updated the body w/ metrics for 5k filters